code/modules/paperwork/paper.dm

```diff
@@ -906,3 +906,8 @@
 
 	info = parsepencode(template, null, null, FALSE)
 #undef MAX_FIELDS
+
+/obj/item/paper/Almayer/CSG
+	name = "Colonial Space Grunts"
+	desc = "A tabletop game based around the USCM, easy to get into, simple to play, and most inportantly fun for the whole squad."
+	info = "<div> <img style='align:middle' src='ColonialSpaceGruntsEZ.png'>"
```

Comment:
Just one more thing - you need to use SSassets link generator or this won't work on the live server due to CDN setup
You can do this by using `SSassets.transport.get_asset_url("name_of_the_file")`
Because this would be done at runtime, you need to set it within an Initialize proc

Reply:
eg instead of just doing info =   you'd do 

```
/obj/item/paper/Almayer/CSG/Initialize(mapload, ...)
  . = ..()
  info = "...."
```